dokka-integration-tests/gradle/src/main/kotlin/org/jetbrains/dokka/it/gradle/AbstractGradleIntegrationTest.kt

```diff
@@ -48,6 +49,12 @@ abstract class AbstractGradleIntegrationTest : AbstractIntegrationTest() {
                     else
                         null,
 
+                    "-Porg.gradle.workers.max=1",
```

Comment:
```suggestion
                    "-Dorg.gradle.workers.max=1",
```

thanks for spotting this @whyoleg!